packages/calypso-products/src/constants/akismet.ts

```diff
@@ -22,6 +22,18 @@ export const PRODUCT_AKISMET_ENTERPRISE_2M_YEARLY = 'ak_ep2m_yearly_1';
 export const PRODUCT_AKISMET_ENTERPRISE_2M_MONTHLY = 'ak_ep2m_monthly_1';
 export const PRODUCT_AKISMET_ENTERPRISE_GT2M_YEARLY = 'ak_epgt2m_yearly_1';
 export const PRODUCT_AKISMET_ENTERPRISE_GT2M_MONTHLY = 'ak_epgt2m_monthly_1';
+export const PRODUCT_AKISMET_PRO_MONTHLY = 'ak_pro5h_monthly';
+export const PRODUCT_AKISMET_PRO_YEARLY = 'ak_pro5h_yearly';
+export const PRODUCT_AKISMET_PRO_BI_YEARLY = 'ak_pro5h_bi_yearly';
+export const PRODUCT_AKISMET_BUSINESS_1K_MONTHLY = 'ak_bus5k_monthly';
+export const PRODUCT_AKISMET_BUSINESS_1K_YEARLY = 'ak_bus5k_yearly';
+export const PRODUCT_AKISMET_BUSINESS_1K_BI_YEARLY = 'ak_bus5k_bi_yearly';
+export const PRODUCT_AKISMET_ENTERPRISE_T1_MONTHLY = 'ak_ep15k_monthly';
+export const PRODUCT_AKISMET_ENTERPRISE_T1_YEARLY = 'ak_ep15k_yearly';
+export const PRODUCT_AKISMET_ENTERPRISE_T1_BI_YEARLY = 'ak_ep15k_bi_yearly';
+export const PRODUCT_AKISMET_ENTERPRISE_T2_MONTHLY = 'ak_ep25k_monthly';
+export const PRODUCT_AKISMET_ENTERPRISE_T2_YEARLY = 'ak_ep25k_yearly';
+export const PRODUCT_AKISMET_ENTERPRISE_T2_BI_YEARLY = 'ak_ep25k_bi_yearly';
```

Comment:
Instead of T1/ T2, could we include the API count instead? 

```
PRODUCT_AKISMET_ENTERPRISE_15K_YEARLY
PRODUCT_AKISMET_ENTERPRISE_25K_YEARLY
```

Reply:
Sure!